src/graph/executor/algo/MultiShortestPathExecutor.cpp

```diff
@@ -0,0 +1,221 @@
+// Copyright (c) 2022 vesoft inc. All rights reserved.
+//
+// This source code is licensed under Apache 2.0 License.
+#include "graph/executor/algo/MultiShortestPathExecutor.h"
+
+#include "graph/planner/plan/Algo.h"
+DECLARE_int32(num_operator_threads);
+namespace nebula {
+namespace graph {
+folly::Future<Status> MultiShortestPathExecutor::execute() {
+  SCOPED_TIMER(&execTime_);
+  pathNode_ = asNode<MultiShortestPath>(node());
+  terminationVar_ = pathNode_->terminationVar();
+
+  if (step_ == 1) {
+    init();
+  }
+
+  std::vector<folly::Future<Status>> futures;
+  auto leftFuture = folly::via(runner(), [this]() { return buildPath(false); });
+  auto rightFuture = folly::via(runner(), [this]() { return buildPath(true); });
```

Comment:
I want to remind you that the `buildPath` will be executed in different threads carefully, don't share the same variables inside the function, especially the fields of the class.

Reply:
Agree, the previous version used local variables but may cause data copying, so use the  member variables